chap-architecture.tex

```diff
@@ -1474,6 +1475,11 @@ \subsubsection{CHERI Concentrate 64-bit format for 32-bit address spaces}
 \vspace{-1.5em}
 \end{figure}
 
+\Cref{fig:cheric64} shows the capability format and decoding method
+for 64-bit CHERI concentrate.  The format contains a 32-bit address,
+$a$, 12 hardware defined permission bits, a 4-bit object type and 15
```

Comment:
I don't know if the 15 bits is correct.  From the diagram it looks like B is 8 bits, T is 6 bits, and IE is 1 bit == 15, but with 16 bits for p + otype, that leaves an extra bit unaccounted for.  In the source it seems that IE is rendered as a bitbox of 2 bits, so is that really 2 bits or is there a 1-bit reserved field missing (that is used on RISC-V for the flags field)?